rickroll.py

```diff
@@ -153,5 +174,55 @@ def original():
 
     return str(play_tune(tune))
 
+@app.route("/sms", methods=["POST"])
+def sms():
+    """Adds an incoming SMS to a queue, to reply to later."""
+    bucket.put_object(
+        Key='queue/{to}/{from_}'.format(to=request.form['To'],
+                                        from_=request.form['From']),
+        Body='',
```

Comment:
Or am I correct that these queued messages get deleted later (when we process them), so storing the SMS received in them is not exactly useful. :)

Would it make more sense to have a separate `logs/` folder, which just logs incoming and outbound messages and timestamps,. but doesn't play a role in message queuing at all? (Absence of a logs directory isn't a blocker for this change, but is certainly a nice-to-have!)